lib/model_patches.rb

```diff
@@ -14,7 +14,7 @@
   #     # any value which is set here
   #     return @default_letter if @default_letter
   #     return nil if self.message_type == 'followup'
-  #     "If you uncomment this line, this text will appear as default text in every message"
+  #     # "If you uncomment this line, this text will appear as default text in every message"
```

Comment:
I don't think you'd want to override the method without this line, so -1 to this commit.